core/src/main/scala/io/jfc/Printer.scala

```diff
@@ -129,13 +129,13 @@ final case class Printer(
       jsonString.span(Printer.isNormalChar) match {
         case (prefix, suffix) =>
           builder.append(prefix)
-          if (suffix.nonEmpty) appendJsonString(suffix, false)
+          if (suffix.nonEmpty) appendJsonString(suffix, normalChars = false)
```

Comment:
Oh, so it looks like it just does these for booleans? I guess it kind of makes sense for clarity…